tests/.eslintrc

```diff
@@ -0,0 +1,2 @@
+{ "env": { "jest": true } }
```

Comment:
Rather than a nested .eslint file, I'd probably suggest adding this to the `overrides` section in our root .eslintrc.js file:

https://github.com/mozilla/blurts-server/blob/03fdd03dabf7cf37a6f60c1e328f5efb761c0cea/.eslintrc.js#L10-L32

Probably something adjacent to:
```js
{
  "files": [
    "tests/**/*.js",
  ],
  "env": {
    "jest": true
  }
}
```

Reply:
Moved into our root `.eslintrc.js`